### DISCUSSION TITLE
wallet core

### DISCUSSION THREAD

**Discussion by guardesign** [trustwallet/wallet-core#1302] [Q&A]
Hi

we are developing a wallet for the exchange project and we want to use the wallet core library.
I wonder to know what's the disadvantage of using Linux for code-wallet SDK !!
it's so critical for us to know that

**Comment by optout21** [comment]
No disadvantage, it's just that wallet-core@Linux is not a production platform for us, we use Linux only for development.  But wallet-core can be used perfectly well on Linux/C++.

**Reply by guardesign** [reply]
Hi 
I hope doing great, unfortunately, we can't build a wallet core in Linux and encounter some errors and unfortunately, we can't find any documentation and something that can solve our problem is there any source or tutorial about this issue !!?
best

**Reply by guardesign** [reply]
and we encounter that problem in MACOS !!!!

**Reply by 0x00trust** [reply]
You can download https://github.com/trustwallet/wallet-core/releases/download/2.6.18/TrustWalletCore-2.6.18.tar.xz
without build again.
Please note that this is core only and you must dev your self wallet for ios or android ( this project have a little sample folder you can find it
https://github.com/trustwallet/wallet-core/tree/master/samples)
This project not full support UI
I'm lose 1-2 hours to build wallet core library with AppCode for Mac Os
https://www.jetbrains.com/objc/download/
If you need UI you can see 
https://github.com/AlphaWallet/alpha-wallet-ios
and
https://github.com/AlphaWallet/alpha-wallet-android
